product-service/serverless.ts

```diff
@@ -63,6 +130,39 @@ const serverlessConfiguration: Serverless = {
             path: '/products',
             cors: true
           }
+        },
+        {
+          sqs: {
```

Comment:
Is it a function that handles HTTP requests or is it a function that processes products in SQS? It _could_ be both, but in _must not_ be both (task requirement!)

Reply:
Updated!